tests/tests_unit/test_approval_modules_snapshots_clean/cdf_oid_example_data.yaml

```diff
@@ -20,8 +20,6 @@ deleted:
   - db_name: asset_oid_workmate
     name:
```

Comment:
Since we are no longer splitting this up into different batches, all tables in the same db is deleted at the same time. Which explains the changes to this snapshot.